airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -133,6 +133,7 @@ def _get_org_repositories(config: Mapping[str, Any], authenticator: MultipleToke
             stream = RepositoryStats(
                 authenticator=authenticator,
                 repositories=unchecked_repos,
+                # This parameter is deprecated and in future will be used sane default, page_size: 10
```

Comment:
Can we add sone unit test for cover backward compatibility.

Reply:
In really I have only removed OPTIONAL param "page_size_for_large_streams" from `spec.json` and nothing more for now.

Hm I am stuck finding how good way to test this case.

Reply:
Attempt to read data using a custom page size or default and then compare the resulting quantity of records with the expected amount.